dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistryFactory.java

```diff
@@ -26,6 +26,7 @@
 
 import java.util.Collection;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;

```

Comment:
remove unused imports

Reply:
ok,i will remove  it.but I do not know why all checks have failed.could you tell me reason? I have never used travis CI before.